charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -116,6 +119,24 @@ spec:
               {{- if .Values.clickhouse.resources }}
               resources: {{ toYaml .Values.clickhouse.resources | nindent 16 }}
               {{- end }}
+            {{- if .Values.clickhouse.backup.enabled }}
+            - name: clickhouse-backup
+              image: {{ template "posthog_backup.clickhouse.image" . }}
+              imagePullPolicy: Always
+              command:
+                - /bin/bash
```

Comment:
Not sure if the `/bin/bash -c` wrapper is needed

Reply:
I left it as it's, clickhouse uses the same wrapper 
https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/templates/clickhouse_instance.yaml#L96